source/OptimaJet.DWKit.StarterApplication/BackendServicesExtension.cs

```diff
@@ -48,6 +48,8 @@ public static IServiceCollection AddApiServices(this IServiceCollection services
                 {
                     builder.AddResource<ProductActions,Guid>("product-actions");
                     builder.AddResource<ProjectToken>("project-tokens");
+                    builder.AddResource<ProductActionRuns, Guid>("product-action-runs");
```

Comment:
do these need to be resources if they aren't jsonapi payloads? (or are they?)

Reply:
They were until last night when I figured out the issues.  I removed them and the tests still pass.  I will change this.